README.md

```diff
@@ -1,3 +1,97 @@
-# tsuru-performance-tests
+# Tsuru Performance test suite
+
+The Tsuru test suite is a set of jmeter-ruby-dsl scripts describing a set of performance testing scenarios.
+They each can be found in the `dsl` folder and run locally or exported to jmx files for running on Blazemeter,
+Flood.io or on your own servers, to provide more resources to a performance test.
+
+Currently, these tests are only runnable against the CI environment.
+
+## How to install
+
+```
+Brew install jmeter
+git clone https://github.com/alphagov/tsuru-performance-testing
+cd ./tsuru-performance-testing
+bundle install
+```
+
+## How to run these tests locally
+
+To run the tests:
+
+```
+cd  ./dsl
+bundle exec ruby deploy_apps.rb <TSURU_ENVIRONMENT_NAME>  <TSURU_TARGET_HOST> 100 100
+```
+
+This will generate a jmeter.jmx file which you can load into the Jmeter gui or use in a headless mode.
```

Comment:
I think we should add how to run jmeter from the console. For instance, something like this:

This will generate a jmeter.jmx file which you can load into Jmeter in GUI mode:

```
jmeter -t jmeter2.jmx
```

Or in the headless mode with `-n` and saving results for later analysis `-l my_results.jtl`
```
jmeter -n -t jmeter2.jmx -l my_results.jtl
```